server/cluster/cluster.go

```diff
@@ -824,6 +824,7 @@ func (c *RaftCluster) HandleStoreHeartbeat(heartbeat *pdpb.StoreHeartbeatRequest
 	}
 	c.core.PutStore(newStore)
 	c.hotStat.Observe(storeID, newStore.GetStoreStats())
+	c.hotStat.ObserveSlowStoreStatus(storeID, newStore.IsSlow())
```

Comment:
It's strange to observe slow store status in hotStat. Maybe we need to change some naming to make it more reasonable.

Reply:
Accept, it's more reasonable use an individual `slowStat` to collect it.